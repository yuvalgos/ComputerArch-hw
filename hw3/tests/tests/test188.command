./dflow_calc tests/test188.dat tests/test188.in d6 d171 p112 p1 d64 d189 d62 d72 d84 d156 d107 p31 d145 d119 p125 d42 p90 p17 d22 p33 p156 d104 p87 d100 d109 p28 d132 p115 d43 p191 p117 p31 p78 d50 p87 d199 d140 p77 p37 d64 p159 d105 p176 d157 p78 p44 d37 p154 d184 d95 p42 p34 p166 p136 d159 p127 d155 d12 d171 p68 d6 d195 d82 d138 p170 p175 p182 p102 p73 d58 d188 d32 p121 p26 p11 p43 d26 p122 p133 p168 p194 p53 d148 p121 p97 p94 d156 p38 d139 d155 p103 d117 d16 d181 p30 p23