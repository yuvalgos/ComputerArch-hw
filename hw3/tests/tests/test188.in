3 22 12 28
4 10 0 19
7 14 21 3
0 15 8 16
5 11 9 12
6 12 1 28
7 2 13 29
5 21 20 8
0 15 27 10
7 15 20 24
4 11 22 20
0 13 4 18
8 18 17 21
6 31 22 9
8 24 23 13
6 9 13 1
1 13 29 19
1 8 9 8
7 23 18 30
4 2 4 8
6 27 26 13
6 28 26 10
6 24 29 30
5 24 11 21
2 17 18 13
0 3 17 19
7 19 21 8
6 21 22 6
6 23 3 11
7 10 27 25
8 15 5 22
4 8 0 4
1 7 31 8
3 25 26 22
8 29 4 12
0 20 23 19
0 0 20 19
8 15 4 7
8 6 5 27
8 11 12 13
0 30 22 19
2 13 11 7
1 6 7 26
3 16 11 18
0 17 22 20
2 15 9 13
4 17 7 21
3 24 30 24
2 0 5 18
4 24 16 25
5 27 14 17
1 31 17 11
3 16 17 27
0 13 9 22
3 0 27 20
4 13 0 10
6 1 23 13
8 25 23 12
0 9 20 16
2 27 22 28
6 3 29 17
2 10 27 29
8 7 21 10
5 26 16 7
7 26 3 21
6 19 22 27
8 9 28 17
7 7 7 11
1 25 18 14
7 3 20 1
2 18 11 14
3 30 19 2
1 25 23 24
8 30 19 20
1 13 9 5
2 21 20 19
8 23 9 23
5 13 3 11
1 5 21 22
3 29 23 3
7 1 23 11
7 8 16 5
8 23 22 28
6 14 29 11
5 10 30 1
8 25 14 16
3 12 23 14
5 7 12 3
6 7 22 7
2 18 6 13
0 24 3 4
6 20 13 27
6 20 10 14
1 13 5 31
6 10 8 29
4 23 6 3
5 23 11 2
8 8 5 9
4 15 18 7
8 3 30 12
8 28 19 30
5 9 26 18
6 10 18 24
1 28 18 8
1 26 15 20
6 26 25 12
0 6 27 6
7 29 7 30
3 1 20 7
1 15 25 4
6 8 10 6
5 16 3 20
3 26 31 12
0 11 22 27
7 2 4 29
5 10 25 30
8 6 8 0
5 3 25 2
6 30 23 23
1 11 17 29
2 18 26 29
3 12 27 17
7 16 23 4
7 17 30 28
7 2 31 26
8 15 31 8
2 6 18 5
0 10 4 16
0 12 30 4
7 13 19 27
0 4 2 4
5 8 13 27
8 3 6 20
4 6 28 26
7 1 4 21
3 14 6 29
5 14 28 27
4 9 28 0
0 1 30 31
6 3 7 30
5 21 13 16
4 11 15 13
5 20 16 10
8 22 19 28
6 9 24 27
0 6 4 31
1 2 11 11
3 19 16 29
4 8 13 6
2 28 21 3
3 7 11 16
4 23 17 31
1 30 15 16
6 20 30 15
1 15 25 10
6 12 27 30
2 14 24 10
0 29 12 4
0 20 10 6
2 7 25 2
7 11 13 27
4 12 1 29
6 12 30 24
7 31 10 9
0 5 5 20
0 16 1 20
1 7 0 23
2 24 7 6
8 15 22 27
3 23 31 5
0 31 23 24
1 13 12 11
8 9 3 21
5 10 6 25
4 7 28 13
7 29 27 17
4 22 29 16
7 31 6 15
8 26 0 4
4 6 9 16
1 12 0 26
6 10 30 7
2 30 20 30
8 8 16 10
7 13 14 11
1 7 17 26
2 17 30 14
6 31 16 15
2 21 4 7
5 2 13 25
3 20 7 22
6 8 21 14
0 13 19 10
8 18 6 13
6 8 20 11
6 28 12 30
7 19 8 9
7 3 21 27
4 31 13 30
6 1 20 9
