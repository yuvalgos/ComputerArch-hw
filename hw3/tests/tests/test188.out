Reading the opcodes latency info from tests/test188.dat ... Got latency for 9 opcodes
Reading the program file tests/test188.in ... Found 200 instructions
getProgDepth()==267
getInstDeps(6)=={-1,-1}
getInstDeps(171)=={162,160}
getDepDepth(112)==147
getDepDepth(1)==0
getInstDeps(64)=={60,27}
getInstDeps(189)=={184,85}
getInstDeps(62)=={27,61}
getInstDeps(72)=={28,49}
getInstDeps(84)=={73,80}
getInstDeps(156)=={90,127}
getInstDeps(107)=={88,73}
getDepDepth(31)==0
getInstDeps(145)=={130,51}
getInstDeps(119)=={46,107}
getDepDepth(125)==168
getInstDeps(42)=={32,-1}
getDepDepth(90)==116
getDepDepth(17)==19
getInstDeps(22)=={-1,-1}
getDepDepth(33)==7
getDepDepth(156)==118
getInstDeps(104)=={98,92}
getDepDepth(87)==147
getInstDeps(100)=={65,73}
getInstDeps(109)=={85,-1}
getDepDepth(28)==43
getInstDeps(132)=={126,92}
getDepDepth(115)==163
getInstDeps(43)=={39,12}
getDepDepth(191)==206
getDepDepth(117)==215
getDepDepth(31)==0
getDepDepth(78)==67
getInstDeps(50)=={2,46}
getDepDepth(87)==147
getInstDeps(199)=={190,172}
getInstDeps(140)=={129,122}
getDepDepth(77)==116
getDepDepth(37)==27
getInstDeps(64)=={60,27}
getDepDepth(159)==205
getInstDeps(105)=={85,86}
getDepDepth(176)==181
getInstDeps(157)=={155,130}
getDepDepth(78)==67
getDepDepth(44)==62
getInstDeps(37)=={-1,32}
getDepDepth(154)==163
getInstDeps(184)=={156,160}
getInstDeps(95)=={42,69}
getDepDepth(42)==27
getDepDepth(34)==17
getDepDepth(166)==167
getDepDepth(136)==137
getInstDeps(159)=={85,146}
getDepDepth(127)==72
getInstDeps(155)=={59,152}
getInstDeps(12)=={-1,7}
getInstDeps(171)=={162,160}
getDepDepth(68)==36
getInstDeps(6)=={-1,-1}
getInstDeps(195)=={180,182}
getInstDeps(82)=={0,21}
getInstDeps(138)=={118,51}
getDepDepth(170)==188
getDepDepth(175)==156
getDepDepth(182)==194
getDepDepth(102)==134
getDepDepth(73)==62
getInstDeps(58)=={35,52}
getInstDeps(188)=={130,185}
getInstDeps(32)=={13,31}
getDepDepth(121)==38
getDepDepth(26)==22
getDepDepth(11)==0
getDepDepth(43)==40
getInstDeps(26)=={7,17}
getDepDepth(122)==53
getDepDepth(133)==154
getDepDepth(168)==160
getDepDepth(194)==208
getDepDepth(53)==19
getInstDeps(148)=={129,145}
getDepDepth(121)==38
getDepDepth(97)==70
getDepDepth(94)==123
getInstDeps(156)=={90,127}
getDepDepth(38)==20
getInstDeps(139)=={88,118}
getInstDeps(155)=={59,152}
getDepDepth(103)==134
getInstDeps(117)=={85,114}
getInstDeps(16)=={-1,-1}
getInstDeps(181)=={152,174}
getDepDepth(30)==7
getDepDepth(23)==18
